howso/engine/trainee.py

```diff
@@ -378,7 +383,7 @@ def active_session(self) -> Session | None:
             The session instance, if it exists.
         """
         if isinstance(self.client, AbstractHowsoClient) and self.client.active_session:
-            return Session.from_openapi(self.client.active_session, client=self.client)
+            return self.client.active_session
```

Comment:
Will still need to create the engine.Session object to return here

Reply:
Addressed by: c28e6de8dded39f04ba3206735fe885f931d8d2e

Reply:
active_session is not the name of the session. Should probally use `from_dict`